jerry-core/lit/lit-char-helpers.c

```diff
@@ -150,8 +150,7 @@ lit_char_is_white_space (lit_code_point_t c) /**< code point */
   }
 
   return (c <= LIT_UTF16_CODE_UNIT_MAX
-          && ((c >= lit_unicode_white_space_interval_starts[0]
-               && c <= lit_unicode_white_space_interval_starts[0] + lit_unicode_white_space_interval_lengths[0])
+          && ((c >= lit_unicode_white_space_interval_starts[0] && c <= lit_unicode_white_space_interval_ends[0])
```

Comment:
`lit_unicode_white_space_interval_starts[0]` is an `u16`,
`lit_unicode_white_space_interval_lengths[0]` is an `u16`,
their sum is an `int`, so a simple cast to `lit_code_point_t` should resolve this warning

Reply:
OK  then

Reply:
There is a lot more msvc compile warnings, I'd like if 
https://github.com/jerryscript-project/jerryscript/issues/4852
https://github.com/jerryscript-project/jerryscript/issues/4853
https://github.com/jerryscript-project/jerryscript/issues/4854
Can be merged, and we can setup the github windows ci to monitoring these warnings